FusionIIIT/applications/globals/api/views.py

```diff
@@ -103,12 +103,6 @@ def profile(request, username=None):
         return Response(data=resp, status=status.HTTP_200_OK)
     elif profile['user_type'] == 'faculty':
         return redirect('/eis/api/profile/' + (username+'/' if username else ''))
-    elif profile['user_type'] == 'staff':
-        resp = {
-            'user' : user_detail,
-            'profile' : profile,
-        }
-        return Response(data=resp, status=status.HTTP_200_OK)
```

Comment:
And why are we removing the case when the user is staff?

Reply:
This piece of code was added by the dashboard module member in my local setup and we forgot to revert it back, he told that these changes should not be here and these changes will be reflected in the PR they would raise after their implementation is completed.